sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/spanner/changestreams/util/TimestampGenerator.java

```diff
@@ -24,7 +24,7 @@
 import java.util.concurrent.TimeUnit;
 
 public class TimestampGenerator extends Generator<Timestamp> {
-  private static final long MIN_SECONDS = Timestamp.MIN_VALUE.getSeconds();
+  private static final long MIN_SECONDS = 0;
```

Comment:
Why do we need to do this?

I think that the generator itself should produce values that are valid within its domain (MIN, MAX). If we need to restrict it, we can configure our test to only allow for values where seconds >= 0.

This way the generator is general and can be used for other use cases that need to test this.

Reply:
Done